Dockerfile

```diff
@@ -240,17 +240,15 @@ ENV DEPENDABOT_NATIVE_HELPERS_PATH="/opt" \
   MIX_HOME="/opt/hex/mix"
 
 USER dependabot
-RUN mkdir -p /opt/bundler/v1 \
-  && mkdir -p /opt/bundler/v2
-RUN bash /opt/bundler/helpers/v1/build /opt/bundler/v1
-RUN bash /opt/bundler/helpers/v2/build /opt/bundler/v2
-RUN bash /opt/go_modules/helpers/build /opt/go_modules
-RUN bash /opt/hex/helpers/build /opt/hex
-RUN bash /opt/npm_and_yarn/helpers/build /opt/npm_and_yarn
-RUN bash /opt/python/helpers/build /opt/python
-RUN bash /opt/terraform/helpers/build /opt/terraform
-RUN bash /opt/composer/helpers/v1/build /opt/composer/v1
-RUN bash /opt/composer/helpers/v2/build /opt/composer/v2
+RUN bash /opt/bundler/helpers/v1/build
```

Comment:
Could it be worthwhile to collapse these into a single `RUN` command and thus create a single FS layer? Admittedly they're not large layers, but may save time when downloading image & building.

Also, I don't think the `bash` is necessary because each is an executable file with a hash-bang?

Reply:
I guess the downside to that would be that they're not cached individually, so changing a single helper requires reinstalling all of them right? Not sure what's the best trade-off in that case

Reply:
Yeah, I was kind of leaving this as-is, some of these installs do more than we'd ideally like - the build script is a bit of a black box in terms of core overall, it _could_ feasibly install some large dependencies ( for example, pulling in and vendoring an old binary for python2, or similar ).

I'd be inclined to keep these layers separate even if it is currently a little ceremonial, but I'd like to do an image optimisation pass more generally as well.

Reply:
> I guess the downside to that would be that they're not cached individually, so changing a single helper requires reinstalling all of them right?

True. With separate layers, this may also cause unneccessary reinstalling since layers are rebuilt if previous layers change. So kinda caught either way in building. The gain for combining would be less in building time and more in downloading time.

> I'd be inclined to keep these layers separate even if it is currently a little ceremonial, but I'd like to do an image optimisation pass more generally as well.

I'm fine saving this for later 👍